src/main/java/com/onkibot/backend/models/UserDetailModel.java

```diff
@@ -13,19 +13,26 @@
   protected UserDetailModel() {}
 
   public UserDetailModel(User user) {
+    this(user, user);
+  }
+
+  public UserDetailModel(User user, User forUser) {
```

Comment:
Users should not be able to see the detailed model of other users, so there's no point in adding the feedback context for the session user.